src/sql/magic.py

```diff
@@ -619,6 +622,73 @@ def _persist_dataframe(
         display.message_success(f"Success! Persisted {table_name} to the database.")
 
 
+def find_toml_path():
+    """
+    Recursively finds an absolute path to pyproject.toml starting
+    from current directory
+    """
+    current = Path().resolve()
+    while not (current / "pyproject.toml").exists():
+        if current == current.parent:
+            return None
+
+        current = current.parent
+    display.message(f"Found pyproject.toml from '{current}'")
+
+    return str(current) + "/pyproject.toml"
```

Comment:
Please use string format instead of `+`

Reply:
since this is a path to a file, you can use pathlib.Path:

```
from pathlib import Path

path = Path(current, "pyproject.toml")
str(path)
```

this is better for compatibility with multiple operating systems

Reply:
Made the change. The return statement is now in `util.py` inside `find_path_from_root()`.